xappt_qt/gui/widgets/tool_page.py

```diff
@@ -235,12 +235,18 @@ def _convert_list(self, param: xappt.Parameter) -> QtWidgets.QWidget:
 
     def update_list_param(self, name: str):
         param: xappt.Parameter = getattr(self.tool, name)
+        error: ErrorLabel = param.metadata['error']
         widget: CheckList = param.metadata.get('widget')
```

Comment:
Rhetorical question - why do I use a `get()` on `widget` but not on `error` (as in `update_tool_param`)? Is there ever a case where there is an error label but no widget? Or a case where `update_*_param` can be called without a widget? I don't think that there is.